content/swiftui/concepts/views/terms/capsule/capsule.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Capsule'
+Description: 'Built-in capsule shape that can be modified as needed.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`Capsule`** is one of SwiftUI pre-built shapes like [rectangle](https://www.codecademy.com/resources/docs/swiftui/views/rectangle), rounded rectangle, [circle](https://www.codecademy.com/resources/docs/swiftui/views/circle), ellipse which can be modified as wanted.
```

Comment:
a rounded rectangle is what we're going to use to describe what the capsule looks like! Then we can made a second sentence that mentions other pre-built shapes.

Reply:
See under overview: https://developer.apple.com/documentation/swiftui/capsule